tests/transforms/test_SimCLRTransform.py

```diff
@@ -0,0 +1,22 @@
+from PIL import Image
```

Comment:
Please rename the file to `test_simclr_transform.py`. It is standard to name the file with the tests the same as the original file but just with a `test_` prefix. We still have some old tests that follow a bad practice of naming files according to their class which we shouldn't do.